cli/src/main/java/org/infinispan/cli/commands/kubernetes/Install.java

```diff
@@ -1,32 +1,47 @@
 package org.infinispan.cli.commands.kubernetes;
 
-import static org.infinispan.cli.commands.kubernetes.Kube.OPERATOR_OPERATORGROUP_CRD;
-import static org.infinispan.cli.commands.kubernetes.Kube.OPERATOR_SUBSCRIPTION_CRD;
-
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Optional;
 
 import org.aesh.command.CommandDefinition;
 import org.aesh.command.CommandResult;
 import org.aesh.command.option.Option;
+import org.aesh.command.option.OptionList;
 import org.infinispan.cli.commands.CliCommand;
 import org.infinispan.cli.impl.ContextAwareCommandInvocation;
-import org.infinispan.cli.impl.KubernetesContextImpl;
-import org.infinispan.commons.util.TypedProperties;
+import org.infinispan.cli.impl.KubernetesContext;
+import org.infinispan.cli.logging.Messages;
+import org.infinispan.commons.util.Version;
 
+import io.fabric8.kubernetes.api.model.GenericKubernetesResource;
+import io.fabric8.kubernetes.api.model.ObjectMeta;
 import io.fabric8.kubernetes.client.KubernetesClient;
 
 /**
  * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
  * @since 12.0
  **/
-@CommandDefinition(name = "install", description = "Installs the operator")
+@CommandDefinition(name = "install", description = "Installs the Operator.")
 public class Install extends CliCommand {
 
-   @Option(shortName = 'n', description = "Select the namespace")
+   @Option(shortName = 'n', description = "Specifies a namespace for the Operator. Defaults to installing the Operator across all namespaces.")
    String namespace;
 
+   @Option(shortName = 'c', description = "Selects the version to install and subscribes to upgrades for that release stream.")
```

Comment:
It might be worth adding that if this is not defined, then the latest stable channel is used.